src/hatch/publish/auth.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+import keyring
```

Comment:
i'm pretty sure @ofek prefers that to be imported only when it's needed in `_read_keyring`.

Reply:
Code updated :)